credsweeper/filters/value_not_part_encoded_check.py

```diff
@@ -33,7 +33,7 @@ def run(self, line_data: LineData, target: AnalysisTarget) -> bool:
             return True
 
         if line_data.line_num == target.line_num \
-                and line_data.line_len == target.line_len \
+                and len(line_data.line) == target.line_len \
```

Comment:
It seems this change is for reducing memory usage.. but as a trade off it seems it can decrease the performance..
Did you compared them?

Reply:
The filter is invoked to rarely compare with AnalysisTarget and line_data.line_len is used only once - no sense for cached property.
Measurement shows less than 1% diff in performance.

Reply:
Okay. Thank you for the details. :+1: